config/api.yml

```diff
@@ -621,6 +621,12 @@
         :identifier: storage_tag
       - :name: unassign
         :identifier: storage_tag
+  :ems_events:
```

Comment:
@rodneyhbrown7 Right, I suggest dropping this change from this PR, then the rest can be merged.  And, we can talk about the right way to expose what you want via the API.